src/reducers/book.js

```diff
@@ -0,0 +1,21 @@
+import actions from '../actions/index';
+
+const { action } = actions;
+const { CREATE_BOOK, REMOVE_BOOK } = action;
+
+const booksReducer = initialState => (state = initialState, action) => {
```

Comment:
🤔 You might be able to simplify this if you pass the initial state through the second argument of `createStore`: https://redux.js.org/recipes/structuring-reducers/initializing-state